tikvrpc/endpoint.go

```diff
@@ -81,8 +81,8 @@ func GetStoreTypeByMeta(store *metapb.Store) EndpointType {
 		if label.Key == EngineLabelKey && label.Value == EngineLabelTiFlash {
 			return TiFlash
 		}
-		if label.Key == EngineLabelKey && label.Value == EngineLabelTiFlashMPP {
-			return TiFlashMPP
+		if label.Key == EngineLabelKey && label.Value == EngineLabelTiFlashCompute {
+			return TiFlashCompute
```

Comment:
Is it backward compatible? What if such a new client connects to an old cluster with `tiflash_mpp` as its engine label?

Reply:
Older version of tiflash's lable is fiexed at compilation time(can only be 'tiflash').
And in new versoin of tiflash(disaggregated tiflash mode), we ensure tiflash lable can only be 'tiflash' or 'tiflash_compute'.

Reply:
e.g. 'tiflash_mpp' is never used.

Reply:
OK.